submissions/krylenger/frogger/app.js

```diff
@@ -0,0 +1,246 @@
+const borders = {
+    topY: 49,
+    rightX: 420,
+    bottomY: 464,
+    leftX: 20 
+}
+
+const tileDimensions = {
+    width: 100,
+    height: 83
+}
+
+const score = document.querySelector('.score');
+const lives = document.querySelector('.lives');
+
+const randomSpeed = () => {
+    return Math.floor(Math.random() * (350 - 100 + 1)) + 100;
+}
+
+//////////  Enemy /////////////////
+
+const Enemy = function() {
+    this.width = 70;
+    this.height = 83;
+    this.speed = randomSpeed();
+};
+
+Enemy.prototype.handleGameOver = function () {
+  if (
+    this.x + this.width > player.x &&
+    player.x + player.width > this.x &&
+    player.y > this.y - this.height &&
+    player.y - player.height < this.y
+  ) {
+    setTimeout(() => {
+      player.lives -= 1;
```

Comment:
This code to me looks too long to be `setTimeout` inline handler. How about extracting it into a function (or maybe several functions)?

Reply:
✅ Done